src/ovirtapi/transform.js

```diff
@@ -288,10 +289,12 @@ const Pool = {
 const Snapshot = {
   toInternal ({ snapshot }: { snapshot: ApiSnapshotType }): SnapshotType {
     return {
-      id: snapshot.id,
+      id: snapshot.id || '',
```

Comment:
~~Why add default values?~~  Ok, I see you're using in the in the API function and need to avoid undefined values in the object causing problems.